zanata-war/src/test/java/org/zanata/arquillian/Deployments.java

```diff
@@ -65,7 +66,9 @@ public static void main(String[] args) {
                 // JavaMelody's ServletFilter/Listener interfere with test
                 // deployments
                 new RejectDependenciesStrategy(false,
-                        "net.bull.javamelody:javamelody-core")).asFile();
+                        "net.bull.javamelody:javamelody-core",
+                        "com.google.collections:google-collections"))
```

Comment:
Please add a comment for this one.  Did you ever work out where the google-collections dependency came from?